docs/version_2.0_differences.md

```diff
@@ -9,11 +9,11 @@ features and behavior differences to take into consideration when updating your
     * [Copy a configuration directory](#copy-a-configuration-directory)
     * [New start and stop goals for testing](#new-start-and-stop-goals-for-testing)
     * [Update Liberty license](#update-liberty-license) 
-    * [New debug server goal](#debug-server)
+    * [New debug goal](#debug)
```

Comment:
I would not expect to change a version_2.0_differences file now that 3.x is released. Changing these is not accurate. In 2.0 debug-server goal existed...not debug goal. And further down it talks about the packageFile default, which no longer exists in 3.x.

Reply:
Ah... these are "differences" between v1 and v2 not v2 and v3.. I missed that.     

I was just aiming to fix dead links.   

See what you think, I redid the PR leaving all the text but pointing the links to the older branch docs, e.g. `https://github.com/OpenLiberty/ci.maven/blob/liberty-maven-2.0/docs/.....`